src/components/Tabs/Tabs.tsx

```diff
@@ -199,6 +199,7 @@ export const Tabs = chakra(
             id={`tabs-previous-icon-${id}`}
             name="arrow"
             size="small"
+            title="left arrow"
```

Comment:
I originally asked for the Icons to be titled "left arrow" and "right arrow", but I think what you accidentally did for the Next button works better.  With that, change this to "previous arrow."